services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -65,6 +67,25 @@ class PermitListResource(Resource, UserMixin):
         help='Whether the permit is an exploration permit or not.')
     parser.add_argument('description', type=str, location='json', help='Permit description')
     parser.add_argument('uploadedFiles', type=list, location='json', store_missing=False)
+    parser.add_argument(
+        'exemption_fee_status_code',
+        type=str,
+        help='Fee exemption status for the mine.',
+        trim=True,
+        store_missing=False,
+        location='json')
+    parser.add_argument(
+        'exemption_fee_status_note',
+        type=str,
+        help='Fee exemption status note for the mine.',
+        trim=True,
+        store_missing=False,
+        location='json')
+    parser.add_argument(
+        'site_properties',
+        type=str,
+        help='{ mine_commodity_code, mine_disturbance_code}.',
```

Comment:
Interesting help? 😜 
Should this be something else?

Reply:
done